pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -2330,6 +2333,33 @@ class FixProcessor {
     _addFix(DartFixKind.LINT_ADD_OVERRIDE, []);
   }
 
+  void _addLintFixReplaceWithCollectionLiteral() {
+    final InstanceCreationExpression instanceCreation =
+        node.getAncestor((node) => node is InstanceCreationExpression);
+    final InterfaceType type = instanceCreation.staticType;
+    final generics = type.typeArguments;
+    final buffer = new StringBuffer();
+    var addGenerics = false;
```

Comment:
Yes but if you do something like final a = new List(); it would replace it with <dynamic>[]